src/attack.c

```diff
@@ -432,7 +432,10 @@ launch_plugin (struct scan_globals *globals, struct scheduler_plugin *plugin,
   if (pid < 0)
     {
       plugin->running_state = PLUGIN_STATUS_UNRUN;
-      ret = ERR_CANT_FORK;
+      if (pid == ERR_NO_FREE_SLOT)
```

Comment:
Update `launch_plugin()` documentation.
Maybe `plugin_launch()` should get a new error parameter so we return pid on success and set and error on error.
This way we do not mix pid value and error value.